packages/core/src/test/codewhispererChat/tools/fsRead.test.ts

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import assert from 'assert'
+import * as vscode from 'vscode'
+import { FsRead } from '../../../codewhispererChat/tools/fsRead'
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const TEST_DIR_NAME = 'fsread-tests'
```

Comment:
same with this one, instead of ignoring the lint rule can you just make it camel case?

Reply:
Fixed.